README.md

```diff
@@ -47,6 +47,12 @@ try {
 }
 ```
 
+*Note:*
+This library is meant to parse JSON while providing good error messages on failure. There is no way it can be as fast as php native `json_decode()`.
+
+Its recommended to parse with `json_decode`, and if that returns `null` parse again with jsonlint to get a proper error message out, something like [we do in Composer](https://github.com/composer/composer/blob/56edd53046fd697d32b2fd2fbaf45af5d7951671/src/Composer/Json/JsonFile.php#L283-L318):
```

Comment:
Rather than saying `when it returns null`, I would say `when it fails` (it is possible to have a `null` return value without being an error when running `json_decode('null')`).

And I would change the end of the sentence to say `as done in Composer` rather than `something like we do in Composer`